src/content/docs/tutorial-customer-experience/bottom-funnel-analysis.mdx

```diff
@@ -0,0 +1,190 @@
+
+<CollapserGroup>
+    <Collapser
+        id="ecommerce-journey"
+        title="Ecommerce user journey"
+    >
+<img
+  title="Ecommerce bottom of the funnel"
+  alt="Ecommerce bottom of the funnel"
+  src={cxBotfaEcommerce}
+/>
+
+The user journey is simplified so you can focus on where the bottom of the funnel begins: at checkout. Most users entering the checkout phase plan to purchase something. Reducing errors and latency from this point onward is more likely to improve conversions than focusing on any other part of the funnel.
+    </Collapser>
+    <Collapser
+        id="insurance-journey"
+        title="Car insurance purchase user journey"
+    >
+<img
+  title="Car insurance bottom of the funnel"
+  alt="Car insurance bottom of the funnel"
+  src={cxBotfaCarInsurance}
+/>
+
+In the example above, you have the user's interest in car insurance as they enter information, but you don't know their intent until they see the quote and continue to proceed.
+    </Collapser>
+</CollapserGroup>
+    </Step>
+    <Step>
+### Distinguish between pages and actions
+
+The final steps of a user's journey is likely to be a mix of full page loads and AJAX calls. You'll need to know all pages and AJAX requests for the next step. If you're not sure which requests are running from the page in question, you can run:
+
+```
+SELECT count(*) FROM AjaxRequest WHERE pageUrl like '%FILTER%' FACET groupedRequestUrl SINCE 1 DAY AGO
+```        
+
+<img
+  title="Distinguish between pages and actions"
+  alt="Distinguish between pages and actions"
+  src={cxBotfaPagesActions}
+/>
+    </Step>
+    <Step>
+### Create a scripted monitor for the bottom of the funnel
+
+Make sure you have a [scripted monitor](/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors/) for each path through the bottom of the funnel. The goal is to make sure your bottom-of-the-funnel services are working around the clock.
+
+For example, you may have a checkout flow that calls a different payment API depending on the customer's payment preferences.        
+    </Step>
+    <Step>
+### Import the bottom-of-the-funnel dashboard
+
+Follow the instructions in the [bottom-of-the-funnel analysis quickstart](https://newrelic.com/instant-observability/customer-experience-bottom-funnel-analysis/8d339aa7-69e3-4efa-abc2-9eba600db91c) to install the dashboard and configure it.
+    </Step>
+    <Step>
+### Capture current performance
+
+<img
+  title="Bottom-of-the-funnel KPI tracking"
+  alt="Bottom-of-the-funnel KPI tracking"
+  src={cxBotfaKpiTracking}
+/>
+
+1. Follow the instructions in our [bottom-of-the-funnel analysis README on GitHub](https://github.com/newrelic/oma-resource-center/blob/main/src/content/docs/oma/value-drivers/customer-experience/use-cases/bottom-of-the-funnel-analysis/README.md).
```

Comment:
Might be nice to give someone a little clue about what they're in for

Reply:
Agreed and done!